### ISSUE TITLE
Search: "Files to include" option is difficult know how to use

### ISSUE THREAD

**Issue by jankrassnigg** [microsoft/vscode#121925]
- VS Code Version:
- OS Version:

Steps to Reproduce:

When you use the "Search" functionality to search in all files, you can click the "..." to additionally be able to specify "Files to include" (and exclude).

Interestingly it uses commas (",") as a separator for the different file types. This is contrary to many other tools, like MSVC, which use semicolons as separators.

I only discovered this through trial and error. Just now I noticed that there is actually a tooltip, that mentions this.

In my opinion it should also allow semicolons, to cater to all the people that are used to this from other tools. Additionally, it would be great, if the available options where better discoverable. For example when I press CTRL+P the main toolbar actually shows example text. It would be great if the search input fields could do the same when you click into an empty line.

Does this issue occur when all extensions are disabled?: Yes

**Comment by roblourens** [comment]
I just added then removed placeholder example text because we found it too visually noisy. I added the tooltip instead. It sounds like you found the tooltip though? This is also described in the docs.

One problem with using semicolon to separate is that I don't have a good way to escape characters, and I also don't really want to respect two separators.

**Comment by jankrassnigg** [comment]
I did find the tooltip eventually. Though literally only hours later. A problem is, that the tooltip only shows when you hover over the input field, but DON'T click on it. Once you click on the field, the tooltip won't show up. You would have to move the mouse outside and back in again. Unfortunately in any typical use case, you would click the field right away and thus no tooltip shows up. I assume that is why I didn't find it for so long.

Instead I tried various separators, and somehow the tooltip never showed up in my trials.

> One problem with using semicolon to separate is that I don't have a good way to escape characters

I don't know what you mean with that. Wouldn't you escape stuff with backslashes?

> and I also don't really want to respect two separators

Why not? It should make the search more intuitive to use for many people.
